pkg/sidecar/configmap/filesystem/filesystem_watcher.go

```diff
@@ -34,7 +34,7 @@ const (
 
 // Monitors an attached ConfigMap volume for updated configuration and calls `configUpdated` when
 // the value changes.
-type configMapWatcher struct {
+type ConfigMapWatcher struct {
```

Comment:
Why are we exporting this?

Reply:
An exported function returns this struct: https://github.com/knative/eventing/pull/1004/files/76007d8782c06a57f2c0a0668f90353d34b26085#diff-b7cf539dbfb8cdbe7cb9106adddd3ec0L50. 

Here is the lint rule: https://github.com/golang/lint/blob/d0100b6bd8b389f0385611eb39152c4d7c3a7905/lint.go#L1351

Reply:
Sorry, I guess I'm confused why there's a need to export that type if it's not used anywhere. But, I see it's actually used in the cmd/fanoutsidecar/main.go

Thanks!!!